doc/new_boilerplate_migration.md

```diff
@@ -781,42 +781,117 @@ UiFactory<FooProps> Foo = uiFunction(
 ); 
 ```
 
-#### With Consumed Props
+#### With Prop Forwarding (fka Consumed Props)
```

Comment:
Maybe this should go somewhere else? ugh i hate deciding where docs should go! @joebingham-wk i need your help lol